### DISCUSSION TITLE
OGP画像にOrganizationsのアイコンも表示してほしい

### DISCUSSION THREAD

**Discussion by yuki549** [increments/qiita-discussions#593] [要望・アイデア・不具合]
他社サービスの例で恐縮ですがZennのOGP画像のイメージです。
Organizationsに設定した記事の場合、所属組織のアイコンもOGP画像に入っていた方がアイキャッチ効果が高いというのと
会社のテックブランディングとしてもQiita投稿を推す理由になるのではと思いました。

**Comment by tanaka-asami415** [comment]
@yuki549 
リクエストありがとうございます🙏

現在の仕組みだとOGP作成時にCDNを利用するコストが発生しているのですが
所属組織のアイコンを表示させる場合、追加でコストがかかってしまいます。
そのため時期をお約束することは難しいのですが、前向きに検討したいと思います！

**Comment by goataka** [comment]
弊社もQiitaを主の書き込み先にしているので、是非対応して頂けるとありがたいです。
先日、記事を読んだ応募者が弊社の記事と気づいて貰えて無かったので、残念に感じました。

**Reply by tanaka-asami415** [reply]
@GOAMI-Takaaki 
リクエストありがとうございます！
アイコンを表示させるには、現在の仕組みを変えるなどの検討も必要なため、すぐには難しそうです。
そのため、まずは Organization名を載せるところからできると良いなと思っています！ 🙏

**Reply by goataka** [reply]
名称表示の対応検討、ありがとうございます。
ご考慮頂けてありがたいです。
引き続きアイコン表示のご検討も宜しくお願い致します。

**Comment by kimkim0814** [comment]
@yuki549 @GOAMI-Takaaki 
お待たせいたしました!

OGP画像について、Organizationを紐付けている場合は
Organization名を載せるように変更いたしました!

また、アイコンについては引き続き対応を検討します🙏

**Reply by goataka** [reply]
ご対応ありがとうございます！　
早々にご対応頂けて、嬉しいです。
引き続き、アイコンも宜しくお願い致します♪

**Comment by P-SiZK** [comment]
@yuki549 @GOAMI-Takaaki
大変長らくお待たせいたしました!

OGP画像にユーザーのアイコンとOrganizationのアイコンが表示されるようになりました！

- https://qiita.com/release-notes/2024-10-31
- https://qiita.com/release-notes/2024-11-20

![image](https://github.com/user-attachments/assets/e3b2f54d-db8d-420b-850a-cf4a8202b6ff)

**Reply by yuki549** [reply]
最高です！ありがとうございます😭

**Reply by goataka** [reply]
おー、ありがとうございます😆
会社の人にも共有します！皆喜びます♪
